tests/virtualization/syscontainer_image_test.pm

```diff
@@ -0,0 +1,133 @@
+# SUSE's openQA tests
+#
+# Copyright © 2017 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: tests running system containers images with libvirt-lxc
+# Maintainer: Cédric Bosdonnat <cbosdonnat@suse.de>
+
+use base "basetest";
+use testapi;
+use strict;
+
+sub run() {
+    select_console 'root-console';
+
+    return if check_var('DISTRI', 'sle') && !get_var('SCC_REGCODE');
+
+    # Check the repositories on the host first
+    assert_script_run("zypper lr");
+
+    my $url    = get_required_var('SYSCONTAINER_IMAGE_URL');
+    my $rootfs = '/tmp/rootfs';
+
+    # Create XML file
+    my $scccredentials_mount = << "EOF";
+    <filesystem type='mount' accessmode='passthrough'>
+      <source dir='/etc/zypp/credentials.d/SCCcredentials'/>
+      <target dir='/etc/zypp/credentials.d/SCCcredentials'/>
+    </filesystem>
+EOF
+    if (!check_var('DISTRI', 'sle')) {
+        $scccredentials_mount = '';
+    }
+
+    my $xml = << "EOF";
```

Comment:
do you know about data folder ? 
Idea is that if you have some file which you need to use in your test you can put it into data folder in same repository and then download it during test with curl . For example take a look here https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/tests/kernel/update_kernel.pm#L165